### ISSUE TITLE
to-öl-075 matches badly on ratebeer

### ISSUE THREAD

**Issue by karlp** [hrafnkell/jg#1]
matches up to some "romy" pils....

**Comment by hrafnkell** [comment]
Yeah... My script doesn't like to öl :) How about a pull request?? :)

**Comment by karlp** [comment]
hah :)

**Comment by karlp** [comment]
Actually spent far too long on this.  problem is that requests sends spaces as +, but ratebeer needs to get them as %20 for the search to work the way you expect.  My first hacks on this result in double encoding.  hatred ensued.

**Comment by hrafnkell** [comment]
Yeah I'm not sure - I checked a query in chrome dev tools and it looks like it's sent with + and no %20.

Meh.
